app-frontend/src/app/components/leafletMap/leafletMap.controller.js

```diff
@@ -37,6 +37,13 @@ export default class LeafletMapController {
         if (changes.proposedBounds && changes.proposedBounds.currentValue) {
             this.map.fitBounds(changes.proposedBounds.currentValue);
         }
+
+        // Add layers to map when a change is detected
+        if (changes.layers) {
```

Comment:
This needs to have `&& changes.layers.currentValue` appended to the condition otherwise this causes a console error on initial page load when `layers` is undefined.